File: GrowthWare_CS/GrowthWareFramework/ModelObjects/MDirectoryProfileCollection.cs
using System;
using System.Runtime.Serialization;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public sealed class MDirectoryProfileCollection : MProfileCollection
	{

		MDirectoryProfileCollection() : base()
		{
			// do nothing
		}
		MDirectoryProfileCollection(SerializationInfo info, StreamingContext context)
			: base(info, context)
		{
			// do nothing
		}
	}
}

File: GrowthWare_VB/GrowthWareFramework/ModelObjects/MDirectoryProfileCollection.vb
Imports System.Runtime.Serialization
Imports GrowthWare.Framework.ModelObjects.Base

Namespace ModelObjects
	<Serializable(), CLSCompliant(True)> _
	Public NotInheritable Class MDirectoryProfileCollection
		Inherits MProfileCollection

		Protected Sub New(ByVal info As SerializationInfo, ByVal context As StreamingContext)
			MyBase.New(info, context)
		End Sub
	End Class
End Namespace